timlubanga/ireport_web
# [Finishes #162109892]add admin dashboard

What does this PR do?
It enables an admin to access redflags and interventions raised and change the status of each
Description of Task to be completed:
login to an admin account
click on intervention link
set intervention
click on redflag link
set redflag

How should this be manually tested

Clone ireport_web into your computer, open the ui folder and run admin.html file using any browser. The admin page should display

**Any background context you want to provide?
This is just a mockup page. Full functionality is yet to be implemented

What are the relevant pivotal tracker stories?
#162109892,#162109905
![screenshot from 2018-11-24 06-04-28](https://user-images.githubusercontent.com/40349882/48963937-fba45900-efae-11e8-9cc8-5cf7fe3fe27f.png)
